package.json

```diff
@@ -20,7 +20,7 @@
     "test": "turbo run test",
     "clean": "turbo run clean && rm -rf node_modules",
     "format": "prettier . --write",
-    "type-check": "tsc --build",
+    "type-check": "tsc --build --pretty",
```

Comment:
Thoughts on adding `"pretty": true,` to the root `polaris/tsconfig.json`?

Reply:
Looks like `"pretty": true` is the default. Going to leave it out of the config and the command flag 👍